starboard/shared/posix/socket_receive_from.cc

```diff
@@ -71,7 +76,9 @@ int SbSocketReceiveFrom(SbSocket socket,
 
     if (IsReportableErrno(errno) &&
         socket->error != sbposix::TranslateSocketErrno(errno)) {
-      SB_LOG(ERROR) << "recv failed, errno = " << errno;
+#if !defined(COBALT_BUILD_TYPE_GOLD)
```

Comment:
But now - this is actually possibly hiding information that was there before ? I'd maybe just leave it as it was here - least risk / side effects.

Reply:
IMHO If removing a log statement from a gold build creates an issue then we have greater things to worry about.

Reply:
Okay, but we don't know what downstream users / systems may rely on those log lines. We are open source - any partner system may be parsing those logs somewhere. I agree ideally nobody should but why create the risk/change if we don't absolutely need it ?

Reply:
Updated to remove the commit with the logging changes.